remix-lib/src/execution/execution-context.js

```diff
@@ -3,7 +3,7 @@ var Web3 = require('web3')
 var EventManager = require('../eventManager')
 var EthJSVM = require('ethereumjs-vm')
 var ethUtil = require('ethereumjs-util')
-var StateManager = require('ethereumjs-vm/lib/stateManager')
+var StateManager = require('ethereumjs-vm/dist/stateManager')
```

Comment:
why ```dist/``` ?

Reply:
seems this is where src file are put during packaging and publishing https://github.com/ethereumjs/ethereumjs-vm/blob/master/package.json#L22